src/bytes.rs

```diff
@@ -1156,6 +1200,10 @@ mod tests {
     }
 
     #[test]
+    // I'm not sure what the platform support is for JIT, but it's probably
+    // not all of them. We really care about making sure it works on x86_64.
+    // We can add more platforms as folks request them.
+    #[cfg(target_arch = "x86_64")]
```

Comment:
I would prefer that it would be enabled wherever it is available rather than hardcoding..

Reply:
which is according to docs is:

```
  ARM 32-bit (v5, v7, and Thumb2)
  ARM 64-bit
  Intel x86 32-bit and 64-bit
  MIPS 32-bit and 64-bit
  Power PC 32-bit and 64-bit
  SPARC 32-bit
```

Reply:
I mean sure, but I don't know what to write here. Do you?

Reply:
I guess I could query PCRE2 at runtime inside the test, and if PCRE2 *believes* JIT is enabled, then we run the test, but otherwise skip it. How does that sound?